UnitsNet/Scripts/UnitDefinitions/Flow.json

```diff
@@ -100,6 +100,18 @@
           "Abbreviations": [ "дм³/мин" ]
         }
       ]
+    },
+    {
+      "SingularName": "MillionStandardCubicFeetPerDay",
+      "PluralName": "MillionStandardCubicFeetPerDay",
+      "FromUnitToBaseFunc": "x*0.326992313",
```

Comment:
I think if you swap these two functions, it will be correct. Multiply in FromBaseToUnitFunc, divide in other.